Orders/Orders.cpp

```diff
@@ -234,4 +245,11 @@ std::ostream& operator<<(std::ostream& os, const OrdersList& ordersList) {
         os << order->getDescription() << "\n";
     }
     return os;
+}
+
+string OrdersList::stringToLog() {
+//get last order added.
+Order o = orders.back();
+string out = "Issued the Order:" + o.getDescription();

```

Comment:
same thing here idk if get description returns a string or a string pointer just check to make sure